extensions/kafka/src/test/java/com/hazelcast/jet/kafka/impl/StreamKafkaPTest.java

```diff
@@ -328,26 +327,22 @@ private void testWithJobRestart(
         for (int i = 0; i < messageCount; i++) {
             kafkaTestSupport.produce(topic1Name, i, String.valueOf(i));
         }
-        sleepAtLeastSeconds(3);
```

Comment:
I think this sleep was there so we are sure that the job starts when the items are already in the topic.

Removing the other sleeps makes sense.

Reply:
I've fixed it by sending messages synchronously  https://github.com/hazelcast/hazelcast/pull/25096/commits/af9612d9a4885bb5acbad4482547a2c217ead834